subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -153,13 +151,37 @@ To use these and similar methods, you need to access the top-level dependencies
 Though it is discouraged, as suggested by the comment in the snippet above, you can also access the configurations of a test suite in a top-level `dependencies` block after creating the test suite.  The configurations used by test suites are normal Gradle configurations, but their exact names should be considered an implementation detail subject to change.  For this reason, configuration via the suite's own `dependencies` block is recommended.
 ====
 
+=== New Strongly-typed Dependencies Block
+The `dependencies` block is using a new API to declare dependencies. This API is very similar to the old API, but it provides full
+typing which makes it easier to understand the relationship between a dependency notation and its `Dependency` sub-type, and
+easier to configure from Kotlin and Java.
+
+This API uses a new type, `DependencyAdder`, to provide more strongly-typed dependency addition. It is returned from getters for
+each configuration type. There is also a new `DependencyFactory` to create new dependencies separately from a `DependencyHandler`.
+It can be injected when needed. It is also the recommended way to create dependencies lazily if needed, as shown below.
```

Comment:
```suggestion
Each configuration type accessor returns a `DependencyAdder`. The `DependencyFactory` creates new dependencies separately from a `DependencyHandler`.
You can inject this type or use it to create dependencies lazily when needed.
```
Suggestion: remove "new" usages for the same reasons as above. Rephrase passive voice to active voice. Remove "as shown below" reference which implies a nonexistent (?) example.